src/freenas/usr/local/sbin/disk_resize

```diff
@@ -167,7 +167,7 @@ sas|scsi)
 nvme)
 	# Get controller and current namespace ID.
 	ctrlr=/dev/${dev%n*}
-	nsid=${dev##nvme*n}
+	nsid=`nvme id-ns /dev/${dev} | awk '/^NVME Identify Namespace/ { print $4 }' | xargs | tr -d ': '`
```

Comment:
Why not `nvme get-ns-id`, that seems to be specially made for it?

Reply:
@amotin - Thank you for pointing it out. I totally missed that. Let me update.

Reply:
Thanks, updated.